boards/nordic/nrf54l20pdk/Kconfig

```diff
@@ -0,0 +1,13 @@
+# nRF54L20 PDK board configuration
+
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NRF54L20PDK_NRF54L20_CPUAPP
+
+config BOARD_ENABLE_DCDC
```

Comment:
@gmarull will your DCDC changes be ready soon?

Reply:
yes, please, wait until https://github.com/zephyrproject-rtos/zephyr/pull/74592 gets merged